packages/web-components/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@prestashopcorp/puik-web-components",
+  "version": "0.0.1",
+  "type": "module",
+  "description": "all web-components are settled here",
```

Comment:
``PUIK available by web components.``

Reply:
ok, package descriptions updated for web-component and components too